locale/en/about/working-groups.md

```diff
@@ -113,27 +113,27 @@ Each language community maintains its own membership.
 * [nodejs-fi - Finnish (Suomi)](https://github.com/nodejs/nodejs-fi)
 * [nodejs-fr - French (Français)](https://github.com/nodejs/nodejs-fr)
 * [nodejs-he - Hebrew (עברית)](https://github.com/nodejs/nodejs-he)
-* [nodejs-hi - Hindi (फिजी बात)](https://github.com/nodejs/nodejs-hi)
+* [nodejs-hi - Hindi (हिन्दी)](https://github.com/nodejs/nodejs-hi)
 * [nodejs-hu - Hungarian (Magyar)](https://github.com/nodejs/nodejs-hu)
 * [nodejs-id - Indonesian (Bahasa Indonesia)](https://github.com/nodejs/nodejs-id)
 * [nodejs-it - Italian (Italiano)](https://github.com/nodejs/nodejs-it)
 * [nodejs-ja - Japanese (日本語)](https://github.com/nodejs/nodejs-ja)
 * [nodejs-ka - Georgian (ქართული)](https://github.com/nodejs/nodejs-ka)
 * [nodejs-ko - Korean (한국어)](https://github.com/nodejs/nodejs-ko)
 * [nodejs-mk - Macedonian (Македонски)](https://github.com/nodejs/nodejs-mk)
-* [nodejs-ms - Malay (بهاس ملايو)](https://github.com/nodejs/nodejs-ms)
+* [nodejs-ms - Malay (بهاس ملايو‎)](https://github.com/nodejs/nodejs-ms)
 * [nodejs-nl - Dutch (Nederlands)](https://github.com/nodejs/nodejs-nl)
 * [nodejs-no - Norwegian (Norsk)](https://github.com/nodejs/nodejs-no)
 * [nodejs-pl - Polish (Język Polski)](https://github.com/nodejs/nodejs-pl)
 * [nodejs-pt - Portuguese (Português)](https://github.com/nodejs/nodejs-pt)
 * [nodejs-ro - Romanian (Română)](https://github.com/nodejs/nodejs-ro)
 * [nodejs-ru - Russian (Русский)](https://github.com/nodejs/nodejs-ru)
 * [nodejs-sv - Swedish (Svenska)](https://github.com/nodejs/nodejs-sv)
-* [nodejs-ta - Tamil (தமிழ்)](https://github.com/nodejs/nodejs-ta)
+* [nodejs-ta - Tamil (தமிழர்)](https://github.com/nodejs/nodejs-ta)
```

Comment:
This is not correct. தமிழர் refers to the people who speak the language தமிழ்.

Reply:
@thefourtheye fixed.